front/app/api/app_configuration/types.ts

```diff
@@ -47,8 +47,8 @@ export type IAppConfigurationSettingsCore = {
   topics_term?: Multiloc;
   topic_term?: Multiloc;
   authentication_token_lifetime_in_days: number;
-  maximum_admins_number: number;
-  maximum_project_moderators_number: number;
+  maximum_admins_number: number | null | undefined;
+  maximum_project_moderators_number: number | null;
```

Comment:
Sorry, forgot to mention that `maximum_project_moderators_number` can also be `undefined`. But it's already handled together with `maximum_admins_number`, so fine to merge.

Reply:
Indeed, let me update it so it is clear in the code for anyone else that might use it